src/org/rascalmpl/util/visualize/replay.html

```diff
@@ -0,0 +1,48 @@
+<!--
```

Comment:
should all these visualisations be moved to a folder inside the parser part? to make clear they are part of the parser visualisation and not generic rascal ones?

Reply:
`replay.html` definitily should. As for the dot stuff, I think it might be useful in other situations too.